src/VRCXDiscordTracker.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace VRCXDiscordTracker
+{
+    internal class VRCXDiscordTracker : ApplicationContext
+    {
+        private readonly NotifyIcon trayIcon;
+        private SettingsForm settingsForm;
+
+        public VRCXDiscordTracker()
+        {
+            // システムトレイアイコンの設定
+            trayIcon = new NotifyIcon()
+            {
+                Icon = Properties.Resources.AppIcon,
+                ContextMenu = new ContextMenu(new MenuItem[]
+                {
+                    new MenuItem("設定", ShowSettings),
+                    new MenuItem("終了", Exit)
```

Comment:
設定画面の項目やメッセージは全部英語なので、コンテキストメニューも英語表記にしちゃっていい気がする。
(※RNGの方も追々…)

```suggestion
                    new MenuItem("Settings", ShowSettings),
                    new MenuItem("Exit", Exit)
```

Reply:
これ、悩んでるというか、i18n対応したほうがいいよなーとは思ってます。